mappings/net/minecraft/client/render/entity/EntityRendererArrow.mapping

```diff
@@ -1 +1,7 @@
 CLASS none/but net/minecraft/client/render/entity/EntityRendererArrow
+	FIELD a ARROW_SKIN Lnone/kp;
```

Comment:
Thinking about it ... maybe they should just be `SKIN` since they're already inside `EntityRenderer<thing>`?